src/middlewared/middlewared/main.py

```diff
@@ -672,15 +678,23 @@ def _call_prepare(
             if methodobj._pass_app['require'] and app is None:
                 raise CallError('`app` is required')
 
+            if methodobj._pass_app['message_id']:
```

Comment:
This puts `message_id` before `app`. There already are methods that use it
```
    @pass_app(message_id=True, rest=True)
    async def update(self, app, message_id, data):
```

Reply:
oops, fixed.